auto-configurations/mcp/spring-ai-autoconfigure-mcp-server/src/main/java/org/springframework/ai/mcp/server/autoconfigure/McpServerAutoConfiguration.java

```diff
@@ -256,6 +256,8 @@ public McpSyncServer mcpSyncServer(McpServerTransportProvider transportProvider,
 
 		serverBuilder.instructions(serverProperties.getInstructions());
 
+		serverBuilder.requestTimeout(serverProperties.getRequestTimeout());
```

Comment:
We need similar change for McpAsyncServer as well?

Reply:
yep this is a miss